docs/source/models/intervention_models/alzheimers/hypothetical_treatment.rst

```diff
@@ -30,4 +30,236 @@
+
+        treat_short -> wane_short
+        wane_short -> no_treat
+
+        treat -> wane 
+
+        wane -> no_treat
+    }
+
+The diagram above illustrates how a simulant should progress through the various testing and treatment related 
+states defined by the client. Each simulant may transition to a new state on each time step. 
+
+Most states have a fixed duration (a multiple of the 
+time step length) where simulants will transition after :math:`\text{duration} / \text{time step}` time steps. 
+The duration is marked in the state node in brackets eg [6 mo]. As desribed in the :ref:`testing intervention <alzheimers_testing_intervention_bbbm>`, 
+some simulants in the `BBBM test eligible` state may transition to tested immediately (low propensity value), some may always self-transition
+(ie, never get tested, high propensity value), and some may self-transition for some number of time steps but eventually transition to tested
+as a result of the time-specific testing rate increasing.
+
+Some states have zero duration, illustrated with a dashed box (rather than the solid ovals for states with nonzero durations). 
+Transitions from a state with zero duration are illustrated with a dashed line. If a simulant transitions to a zero-duration state 
+on a time step, they should also immediately continue to the next state during that same time step, as a part of the same transition.
+
+For example, a simulant in `BBBM test eligible` who is tested and moves to `BBBM test received` would then immediately move to one of 
+that state's two sinks, and would even move directly to another state during the same transition/ time step on a positive test. 
+
+Below are tables with details on how to model these states and transitions, and necessary data values. 
+The value of :math:`i_{MCI}` in the :ref:`cause model <2021_cause_alzheimers_presymptomatic_mci_transition_data_table>` is now updated
+to be equal to :math:`h_{adj} = h_{MCI} \cdot R_h`.
+
+.. _alzheimers_intervention_treatment_data_table:
+
+.. list-table:: Data values and sources
+  :widths: 15 15 30 15
+  :header-rows: 1
+
+  * - Variable
+    - Definition
+    - Source or value
+    - Notes
+  * - :math:`\text{prop}_I`
+    - Simulant lifetime treatment "initiation propensity"
+    - Drawn uniformly from :math:`[0,1)`
+    - Lower value means more likely to initiate testing. Independent from testing propensities.
+  * - :math:`I`
+    - Time- and location-specific treatment initiation rate
+    - Lilly: "The percent of patients with a positive BBBM test who initiate treatment will vary by location and over time – but will not vary by age or sex. In the US: 30% of eligible patients initiate (constant 2030-2100); Japan: 80% of eligible patients initiate (constant 2030-2100); all other countries: 40% of eligible patients initiate in 2030, increasing linearly to 70% by 2035, remaining constant at 70% until 2100.""
+    - 
+  * - :math:`h_{adj}`
```

Comment:
Request from engineers that h_adj and h_MCI not be in the table --- they are perhaps worth describing in a paragraph above or below the table, but they are not central enough to the treatment data values and sources to be included in the table.  (But this can be for future documentation, and does not need to block merging these docs).